### PR TITLE
Corrected some minor bugs in layout and bootstrap-use

### PR DESCRIPTION
**Pull request by fierwebdesign**
It might be just because of some fault on my part, but I had to fix some minor bugs before I got the module working. Thank you Chris, for your great work in the yii-community!

### CODE REVIEW

File: components/AuthBehavior.php [hunk 1]
```diff
@@ -14,6 +14,9 @@
  */
 class AuthBehavior extends CBehavior
 {
+
+	public $admins;
```
Comment (magefad) [inline components/AuthBehavior.php:18]: ?

Comment (fierwebdesign) [inline components/AuthBehavior.php:18]: I got an error that admins was not defined in this behavior. It was solved by inserting this variable declaration. What did I miss?

Comment (magefad) [inline components/AuthBehavior.php:18]: moved to AuthWebUser again by Crisu83

**Comment by cniska** [comment]
Yeah sorry for the hassle, I was implementing yii-auth in a larger project that I was working on and I discovered some issues with the design. I corrected them a bit quickly so I introduced some bugs as well. Sorry for the inconvenience.

**Pull request closed without merging**